app/tests_data.py

```diff
@@ -1,90 +1,66 @@
+# pylint: disable=missing-docstring
+# pylint: disable=invalid-name
 
-from models import *
+from app.models import Ingredient, Recipe, RecipeNutrient, GroceryItem,\
+        GroceryItemIngredient, Tag, TagIngredient, TagRecipe,\
+        TagGroceryItem, IngredientNutrient, RecipeIngredient
```

Comment:
Because of the long list of imports, it may be better to simply import models and reference its classes through "models".

```
from app import models

...

data = models.Ingredient(...)
```